.dockerignore

```diff
@@ -0,0 +1 @@
+#**.env
```

Comment:
Why is this commented out?
I think `**/.env` is a very sensible entry here, as we don't want any secrets ending up baked into the image

We might want to add some extra things, for example:
```
**/__pycache__
**/.pytest_cache
Dockerfile
```